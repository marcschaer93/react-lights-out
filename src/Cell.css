.Cell {
  height: 100px;
  width: 100px;
  background-color: darkgray;
  border: solid 1px black;

  cursor: pointer;
}

.Cell-lit {
  background-color: white;
}
