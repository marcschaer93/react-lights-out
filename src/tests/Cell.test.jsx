import { render } from "@testing-library/react";
import Cell from "../Cell";

// rendering a cell Cell properly: this is a straightforward test, and could even be a snapshot test

// smoke Test
it("should render without crashing", () => {
  render(<Cell />);
});

// snapshot Test
it("should match snapshot", () => {
  const { asFragment } = render(
    <Cell isLit={false} flipCellsAroundMe={10 - 7} />
  );
  expect(asFragment()).toMatchSnapshot();
});
