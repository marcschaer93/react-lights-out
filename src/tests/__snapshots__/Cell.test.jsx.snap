// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot 1`] = `<DocumentFragment />`;
